### ISSUE TITLE
Moving to processors 7.4.1?

### ISSUE THREAD

**Issue by EgoLaparra** [clulab/eidos#414]
The temporal_attachment branch needs version 7.4.1 of processors to get access to norms. Is master going to be updated to this version at some stage?

I have noticed that version 7.4.1 produces a slight different tokenization in some cases what can cause some tests to fail. For example, for the sentence (TestDoc3, paragraph 14):

```
Although, the short-and long-term rainfall forecasts are favorable for agricultural areas 
of the western and central highlands, production of most crops is likely to be limited 
due to insecurity and lack of availability and/or access to farm inputs due to ongoing 
conflict.
```
processors 7.3.1 tokenizes "and/or" as a single token, while 7.4.1 splits it into 3 different tokens.

**Comment by kwalcock** [comment]
I think we were just waiting for a good enough reason and time to switch.  The branch proc740 is now so old that it has complex merge conflicts.  I was going to reapply the changes to a new branch of master.  There was at least one test that would fail last time I looked.  Is it time to do this and should we go for 7.4.1 right away?  I suspect that you already have a good candidate branch.

**Comment by MihaiSurdeanu** [comment]
Yes, let’s release 7.4.1 and update!

**Comment by marcovzla** [comment]
The portuguese2 branch is already using processors 7.4.2
maybe we could use that one?

**Comment by MihaiSurdeanu** [comment]
First of all, @marcovzla: turn off your computer :)

7.4.2 hasn't been released yet. We need to commit new PT models, which will probably happen when @gcgbarbosa is back.

**Comment by marcovzla** [comment]
:} ok i will turn it off.
i merged master into portuguese2 so i think we will be ready to merge once the models are ready.

**Comment by marcovzla** [comment]
Also, seems like this was solved by #415 
should we close this issue?
